src/components/lib/Questions.tsx

```diff
@@ -37,7 +36,7 @@ export abstract class Questions {
    * @param props
    * @returns
    */
-  static getRadio(answer: string, props: IQuestionData): ReactNode {
+  private static getRadio(answer: string, props: IQuestionData): JSX.Element {
```

Comment:
Just curious why you switched from ReactNode to JSX.Element?

Reply:
JSX.Element is the more specific type. ReactNode is sort of a catch all type for when you have an ambiguous return (could be a string, number, node or collection of nodes). I was able to constrain it; hence the simpler type.